jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -137,8 +137,12 @@ class java_bytecode_parsert:public parsert
     }
   }
 
-  u8 read_bytes(size_t bytes)
+  template <typename T>
```

Comment:
commit message "Make read_bytes less likely to be called incorrectly" truncated (larger than... what?)